eth/tracers/internal/tracetest/testdata/call_tracer_parity/suicide.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
How did you make this test data? Should we consider including any test creation tools in the repo?

Reply:
By using [this example code](https://github.com/etclabscore/core-geth/blob/d7543b9c21631466226e0f931edca13f92f9acca/eth/tracers/internal/tracetest/calltrace_test.go#L49-L96), but with the tracer name I need the test for.

Reply:
Ah, looks great. Sorry I didn't see that earlier.

Reply:
I even missed it for long time mate :)